lib/frontend/app.js

```diff
@@ -105,27 +103,30 @@ module.exports = (appTokenGenerator) => {
     res.redirect(info.external_url)
   })
 
-  const sentryMiddleware = async (err, req, res, next) => {
+  const addSentryContext = async (err, req, res, next) => {
     Sentry.withScope(async scope => {
-      scope.setTag('jiraHost', req.session.jiraHost || req.query.xdm_e)
-
-      if (req.session.githubToken) {
-        try {
-          const { data: { login } } = (await res.locals.github.users.get())
```

Comment:
Removing this because, if I understand it correctly, it would make an HTTP request to GitHub for each exception raised.